spock/plugins/helpers/inventory.py

```diff
@@ -144,6 +145,16 @@ def __init__(self, ploader, settings):
         # stores the last click action for confirmation
         self.last_click = None
 
+        # to know when the inventory got synced with server
+        self.is_synchronized = False
```

Comment:
Is there a situation where can this be defined? Should it be set to ``False`` in the class definition or initializer?